### ISSUE TITLE
[GVT] The color of hint message could not be seen clearly on FF

### ISSUE THREAD

**Issue by panlei080988** [IBM/carbon-components-react#1935]
## Environment

> Operating system : Win 10

> Browser : Firefox ESR 52.9.0

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Create an Internet Service on cloud
2. Click Reliability-> Global Load Balancers item
3. Click on Create pool button

## Additional information

- Screenshots or code
![image](https://media.github.ibm.com/user/25201/files/76bdfe00-36c9-11e9-90cf-d10c7ba7ad08)
